util/design/gen-otp-mmap.py

```diff
@@ -60,14 +59,10 @@ def main():
             log.warning('Commandline override of seed with {}.'.format(
                 args.seed))
             config['seed'] = args.seed
-        # Otherwise, we either take it from the .hjson if present, or
-        # randomly generate a new seed if not.
+        # Otherwise we make sure a seed exists in the HJSON config file.
```

Comment:
Makes sense to be explicit. Note that topgen has the same behavior - so it probably makes sense to remove it there as well:
https://github.com/lowRISC/opentitan/blob/5aff873861954239d2bc241bed747d939391e51b/util/topgen.py#L1171-L1178

Reply:
yes, I was planning on addressing that in a follow up PR (i.e., aligning all these scripts to do the same thing here)

Reply:
Fixed in #18793.